app/backend/src/app.py

```diff
@@ -60,7 +61,12 @@
 
 logging.getLogger("couchers.jobs.worker").setLevel(logging.INFO)
 
-start_http_server(port = 8000, registry = main_process_registry)
+if config.config["SENTRY_ENABLED"]:
+    """Sends exception tracebacks to Sentry, a cloud service for collecting exceptions"""
+    sentry_sdk.init(config.config["SENTRY_URL"], traces_sample_rate=0.0)
```

Comment:
What's the `traces_sample_rate`?

Reply:
It is a setting for sending traces to Sentry for their performance monitoring. We already have our own traces and monitoring in grafana, so I would suggest not using that feature for now and setting it to 0. We would also probably generated too many traces to fit in the free tier, so that would make the Sentry performance monitoring somewhat useless.